### DISCUSSION TITLE
How to install wezterm in arch linux

### DISCUSSION THREAD

**Discussion by RiteshChepuri** [wezterm/wezterm#5322] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

wayland

### WezTerm version

wezterm 20240405-180910-cce706b

### Ask your question!

I installed wezterm by downloading wezterm git and makekg -si 
its succesfully installed but i cant seem to openit . What am i doing wrong?

**Comment by ramyjaber1** [comment]
Same problem on arch linux wayland

**Reply by RiteshChepuri** [reply, marked as answer]
in wezterm config you have to include this
enable_wayland=false

it fixes the problem

**Comment by PerseweratywneImponderabilia** [comment]
Appears to be fixed on the main branch.
The issue (at least for me) is gone when building the project from sources.

**Comment by amiramay** [comment]
I installed wezterm using pacman and had the same issue. But, @ramyjaber1 's solution worked. Thanks!
